service/service.go

```diff
@@ -234,6 +234,9 @@ func (svc *service) GetEventPublisher() events.EventPublisher {
 }
 
 func (svc *service) GetLNClient() lnclient.LNClient {
+	if svc.isLNClientShuttingDown {
```

Comment:
is this LNClient specific? or can it be a global boolean indicating the shutdown?

Reply:
I do not think we need one. See https://github.com/getAlby/hub/pull/605